README.md

```diff
@@ -12,16 +12,20 @@ If you have `git` you can clone this repository with the command
 git clone https://github.com/eth-cscs/DLA-Future.git
 ```
 
-Otherwise you can download the archive of the latest `master` branch as a [zip](https://github.com/eth-cscs/DLA-Future/archive/master.zip) or [tar.gz](https://github.com/eth-cscs/DLA-Future/archive/master.tar.gz) archive.
+Otherwise you can download the [releases](https://github.com/eth-cscs/DLA-Future/releases).
```

Comment:
Possibly swap this with the `git clone` if we want to emphasize using released versions, but I think either way is fine.